include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -551,7 +547,8 @@ namespace internal
         case 3:
           for (unsigned int c = 0; c < n_components; c++)
             {
-              if (integrate_values == true && integrate_gradients == false)
+              if (integration_flag & EvaluationFlags::values &&
```

Comment:
```suggestion
              if ((integration_flag & EvaluationFlags::values) &&
```
I don't think this is necessary but I can never remember the order of precedence for `&` and `&&`.